plugins/templates-maven-plugin/src/main/java/com/google/cloud/teleport/plugin/maven/TemplatesReleaseMojo.java

```diff
@@ -140,6 +150,27 @@ public class TemplatesReleaseMojo extends TemplatesBaseMojo {
   @Parameter(defaultValue = "true", property = "generateSBOM", readonly = true, required = false)
   protected boolean generateSBOM;
 
+  @Parameter(
+      defaultValue = "false",
+      property = "publishYamlBlueprints",
+      readonly = true,
+      required = false)
+  protected boolean publishYamlBlueprints;
+
+  @Parameter(
+      defaultValue = "yaml/src/main/yaml",
+      property = "yamlBlueprintsPath",
+      readonly = true,
+      required = false)
+  protected String yamlBlueprintsPath;
+
+  @Parameter(
+      defaultValue = "yaml-blueprints",
+      property = "yamlBlueprintsGCSBucket",
```

Comment:
Are we always uploading to the top level of the GCS bucket ? May be this should be a GCS path ?

Reply:
I believe we are.  We have the standard: gs://<bucketName>/<stagePrefix>/<yamlBlueprintsGCSBucket>/<fileName>

So you would rather give a path at that point, so something like this that could be placed anymore under stagePrefix?
gs://<bucketName>/<stagePrefix>/<yamlBlueprintsGCSBucketPath>/<fileName>

Thanks